### ISSUE TITLE
How can i make my modal dialog be on center position

### ISSUE THREAD

**Issue by dezguell** [craftpip/jquery-confirm#303]
//function//
function ShowContentDialog_Multicolor(titulo, color, url) {
    $.dialog({
        title: titulo,
        content: 'url:' + url,                     
        type: color
    });
}
//Default config//
jconfirm.defaults = {
    title: 'Hello',
    titleClass: '',
    type: 'default',
    draggable: false,
    alignMiddle: true,
    typeAnimated: true,
    content: 'Are you sure to continue?',
    buttons: {},
    defaultButtons: {
        ok: {
            action: function () {
            }
        },
        close: {
            action: function () {
            }
        },
    },
    contentLoaded: function (data, status, xhr) {
    },
    icon: '',
    lazyOpen: false,
    bgOpacity: null,
    theme: 'light',
    animation: 'zoom',
    closeAnimation: 'scale',
    animationSpeed: 400,
    animationBounce: 1.2,
    rtl: false,
    container: 'body',
    containerFluid: false,
    backgroundDismiss: false,
    backgroundDismissAnimation: 'shake',
    autoClose: false,
    closeIcon: null,
    closeIconClass: false,
    watchInterval: 100,
    columnClass: 'small',
    boxWidth: '50%',
    scrollToPreviousElement: true,
    scrollToPreviousElementAnimate: true,
    useBootstrap: true,
    offsetTop: 50,
    offsetBottom: 50,
    dragWindowGap: 15,
    bootstrapClasses: {
        container: 'container',
        containerFluid: 'container-fluid',
        row: 'row',
    },
    onContentReady: function () { },
    onOpenBefore: function () { },
    onOpen: function () { },
    onClose: function () { },
    onDestroy: function () { },
    onAction: function () { }
};

....The image is no related to de code, is just an example.....
![image](https://user-images.githubusercontent.com/30419218/28543791-5b6b6880-7097-11e7-9afa-92d66df7f022.png)

**Comment by craftpip** [comment]
looks like you're not using bootstrap, 
so set `useBootstrap: false` and you're ready to go.

**Comment by dezguell** [comment]
I already did what you suggest, but didn't work. Now the modal is shown in the center of the screen but taken the entire page dimension. So I had to modify the CSS file to set de margin-left & margin-right with a value to allow the dialog be centralized. That solve my problem for a while but that I realy need is that the size of the dialog ajust itself to his content and that it be localized in the center os the page.

//---------CSS code--------//
.jconfirm .jconfirm-scrollpane {
        position: fixed;
        top: 0;
        left: 0;
        right: 0;
        bottom: 0;
        overflow-y: auto;
        -webkit-perspective: 500px;
        perspective: 500px;
        -webkit-perspective-origin: center;
        perspective-origin: center;
        margin-left: 38.33%;
        right: 35%;
    }

//-----Js function-------//
function ShowContentDialog_Multicolor(titulo, color, url) {
    $.dialog({
        title: titulo,
        content: 'url:' + url,                     
        columnClass: 'col-md-6 col-md-offset-3', /*(this do nothing)*/
        type: color
    });
}

//-------Default config-------//
jconfirm.defaults = {
title: 'Hello',
titleClass: '',
type: 'default',
draggable: false,
alignMiddle: true,
typeAnimated: true,
content: 'Are you sure to continue?',
buttons: {},
defaultButtons: {
ok: {
action: function () {
}
},
close: {
action: function () {
}
},
},
contentLoaded: function (data, status, xhr) {
},
icon: '',
lazyOpen: false,
bgOpacity: null,
theme: 'light',
animation: 'zoom',
closeAnimation: 'scale',
animationSpeed: 400,
animationBounce: 1.2,
rtl: false,
container: 'body',
containerFluid: false,
backgroundDismiss: false,
backgroundDismissAnimation: 'shake',
autoClose: false,
closeIcon: null,
closeIconClass: false,
watchInterval: 100,
columnClass: 'small',
boxWidth: '50%',
scrollToPreviousElement: true,
scrollToPreviousElementAnimate: true,
useBootstrap: false,
offsetTop: 50,
offsetBottom: 50,
dragWindowGap: 15,
bootstrapClasses: {
container: 'container',
containerFluid: 'container-fluid',
row: 'row',
},
onContentReady: function () { },
onOpenBefore: function () { },
onOpen: function () { },
onClose: function () { },
onDestroy: function () { },
onAction: function () { }
};
